HarjotSuiingh.md

```diff
@@ -0,0 +1,65 @@
+
+---
+
+name: @HarjotSuiingh
+
+project: E-ink Smart Display
+
+---
+
+
+# Project Name
+
+E-ink Smart Display
+
+
+## Summary
+
+
+What are you going to build? What does it do? Why are you excited to build it?
+
+
+I am going to build an e-ink smart display which is powered by a Rasberry pi. It will be able to act as a display where I can view things such as my calendar and control smart home items that I have whilst also acting as an e-reader making it an all in one device. I am excited for this build because it will allow me to have one device for which I can do everything I need to do which would just make my life simpler.
+
+
+## Plan
+
+
+What steps are you going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+
+I will be using the Pi at the start to get everything set up since it is faster and easier to access over ssh. Then I will be moving the project over to the Pico so its permanently connected to thr Screen.
+
+The sensors+accessories will be used to add functionality to the app and allowing me to make it more personalised based on my location and environment
```

Comment:
could you please expand a little more on this? what type of sensors are you thinking of using and how do you plan on incorporating it in your project?

Reply:
@bellesea I have decided to replace the sensors kit with a Rasberry pi starter kit instead. Using items such as the wires in this kit will help me to add other features into the screen such as for example wiring a set of push buttons to the pi with some jumper wires which links in with the temperature control screen feature in the kindle allowing me to change the colour and brightness of the screen with a press of the buttons.

Reply:
@HarjotSuiingh hmm i'm struggling a little bit to picture the role of the various materials in this kit...

what's the temperature control screen feature in the kindle? and, how do you plan on making it such that you can use a physical control to affect something internal to the kindle? also, to do that you probably only need a potentiometer with some wires which is a lot cheaper haha...

btw, we only offer up to $250 grants per person so we'd ask that you cover the excess yourself...

thank you sm!

Reply:
The sensors include a button kit which will be connected to the pi so i can control the modded firmware's temperature sensors with it. Also, there is no excess money needed as of now! @bellesea

Reply:
All the sensors will be useful for the project, since the modded firmware can show pretty much anything i want, i want to work with connecting different sensors to it and getting information based on the environment around me.

Reply:
ahh cool, thanks! and my bad on the budget, i forgot it was in AUD haha.